pkg/utils/grpcutil/grpcutil.go

```diff
@@ -147,7 +148,16 @@ func GetClientConn(ctx context.Context, addr string, tlsCfg *tls.Config, do ...g
 	if err != nil {
 		return nil, errs.ErrURLParse.Wrap(err).GenWithStackByCause()
 	}
-	cc, err := grpc.DialContext(ctx, u.Host, append(do, opt)...)
+	backoffOpts := grpc.WithConnectParams(grpc.ConnectParams{
+		Backoff: backoff.Config{
+			BaseDelay:  100 * time.Millisecond,
```

Comment:
I found the BaseDelay is 1s in https://github.com/tikv/pd/blob/e199866f59e22e3759a8e9459ef33d57f784890d/pkg/syncer/client.go#L117.
Do we need keep them same?

Reply:
ok